pkg/fanal/analyzer/language/analyze.go

```diff
@@ -104,6 +104,7 @@ func toApplication(fileType types.LangType, filePath, libFilePath string, r xio.
 		pkgs[i].DependsOn = deps[pkg.ID]
 		pkgs[i].Digest = d
 		pkgs[i].Indirect = isIndirect(pkg.Relationship) // For backward compatibility
+		NormalizeLicenses(pkgs[i].Licenses)
```

Comment:
In analyze.go:107-111:

```
NormalizeLicenses(pkgs[i].Licenses)

for j, license := range pkg.Licenses {
    pkgs[i].Licenses[j] = licensing.Normalize(license)
}
```

The NormalizeLicenses() call on line 107 normalizes the licenses, then lines 109-111 iterate and normalize again. This seems redundant.

Reply:
I remember removing this line.
It’s possible that I accidentally reverted this change.